lib/SecretsCrypto/sf-secrets.h

```diff
@@ -31,6 +36,119 @@ typedef enum SfSecretsPluginState_ {
 	SF_SECRETS_PLUGIN_STATE_PLUGIN_SUPPORTS_SET_LOCK_CODE = 1 << 2,
 } SfSecretsPluginState;
 
+typedef enum SfSecretsError_ {
```

Comment:
Could we somehow avoid duplicating the values like this?

Reply:
Techincally we could share the enums with C++ with a little moving around, but as C does not have namespaces, it becomes problematic. And also quite commonly enum values in C are written in UPPER_WITH_UNDERSCORES (in fact a requirement if GObject introspection is wanted).

What could be done is some preprocessing script that mangles the names from the C++ headers.

Reply:
IMO it's not worth the trouble - since these values are part of the D-Bus interface, they are not going to change in the future. Once synchronized, these values will never go out of sync.

Reply:
The DBus interface can change freely, no stability guarantees are provided for it.  But the values defined in the C++ header will not, as I intend to provide BC/SC guarantees for the C++ client library.  In any case, that doesn't change the fact that you're right: they're not going to change.